File: back/KaemeWebApi/Dominio/Models/PedidoModels/PedidoCreate.cs
namespace Dominio.Models
{
    public partial class PedidoCreate
    {

        public Pedido Pedido { get; set; }
        public List<PedidoPeca> Pecas { get; set; }
        public PedidoCobranca Pedido_Cobranca { get; set; }
        
        // Apenas para visualização
        public List<Peca>? PecasReturn { get; set; }

    }
}

File: front/kaemeweb/src/app/modules/shared/models/PedidoModels/pedido-create.ts
import { Peca } from "../peca";
import { Pedido } from "./pedido";
import { PedidoCobranca } from "./pedido-cobranca";
import { PedidoPeca } from "./pedido-peca";

export interface PedidoCreate {
    pedido: Pedido;
    pecas: PedidoPeca[];
    pedido_Cobranca: PedidoCobranca;

    // Apenas para visualização
    pecasReturn?: Peca[];
}
